code/students-performance/src/test/java/ru/unn/agile/StudentsPerformance/Model/PerformanceCalculatorTest.java

```diff
@@ -0,0 +1,219 @@
+package ru.unn.agile.StudentsPerformance.Model;
+
```

Comment:
Please check names of tests: some of them has outdated information (Number)

Reply:
Fixed